recipes/editline/all/conanfile.py

```diff
@@ -47,13 +39,23 @@ def configure(self):
             del self.options.fPIC
         del self.settings.compiler.libcxx
         del self.settings.compiler.cppstd
+
+    def requirements(self):
+        if self.options.terminal_db == "termcap":
+            self.requires("termcap/1.3.1")
+        elif self.options.terminal_db == "ncurses":
+            self.requires("ncurses/6.2")
+
+    def validate(self):
         if self.settings.os == "Windows":
             raise ConanInvalidConfiguration("Windows is not supported by libedit (missing termios.h)")
+        if self.options.terminal_db == "tinfo":
+            # TODO - Add tinfo when available
+            raise ConanInvalidConfiguration("tinfo is not (yet) available on CCI")
```

Comment:
`tinfo` is part of ncurses.
ncurses can be built with tinfo built inside `libncurses.so`, or separately as `libtinfo.so` and `libncruses.so`.
https://github.com/conan-io/conan-center-index/blob/6704c4d14b626b69e74ff5f9c79ed235316fb8b7/recipes/ncurses/all/conanfile.py#L27

Reply:
I see. I didn't change this part, juste moved the verification in validate(). It can be fixed in another PR.

Reply:
What do you sugget? Remove tinfo in possibles values of `terminal_db` option?

Reply:
I would remove (or deprecate) the option.
And pass `tinfo` if `ncurses:with_tinfo` is True, else `ncurses`.

Reply:
editline doesn't have such option. It tests existence of ncurses, then curses, then termcap, then tinfo, that's all.

```autoconf
AC_CHECK_LIB(ncurses, tgetent,,
  [AC_CHECK_LIB(curses, tgetent,,
    [AC_CHECK_LIB(termcap, tgetent,,
      [AC_CHECK_LIB(tinfo, tgetent,,
        [AC_MSG_ERROR([libncurses, libcurses, libtermcap or libtinfo is required!])]
      )]
    )]
  )]
)
```

Reply:
I guess that will work as the `tgetent` symbol will not be found in `ncurses`.
So only removing/deprecating the option value is enough.

Reply:
we can remove it without deprecation, it was an invalid configuration anyway.

Reply:
good call :)